lib/std/special/compiler_rt/floatfmodl_test.zig

```diff
@@ -0,0 +1,15 @@
+const std = @import("std");
+const fmodl = @import("floatfmodl.zig");
+const testing = std.testing;
+
+fn test_fmodl(a: f128, b: f128, exp: f128) !void {
+    const res = fmodl.fmodl(a, b);
+    try testing.expect(exp == res);
+}
+
+test "fmodl" {
+    try test_fmodl(6.8, 4.0, 2.8);
```

Comment:
what about the edge cases of float stuff, ie Nan, +-inf, denormalized numbers, +-0 ?
are they handled?
adding 2 negative numbers for the mod also would be helpful.

if you want to go the long road, you can add one or two bigger numbers.

Reply:
Done in 834c9e25